### ISSUE TITLE
'iotedge check' fails on Nvidia Jetson NX when checking daemon management endpoint

### ISSUE THREAD

**Issue by Chad-McQuillen** [Azure/iotedge#3948]
## Expected Behavior
'iotedge check' should not fail when checking for the daemon management endpoint.

## Current Behavior
```
× config.yaml has correct URIs for daemon mgmt endpoint - Error
    One or more errors occurred. (Cannot assign requested address var/run/iotedge/mgmt.sock)
        caused by: docker returned exit code: 1, stderr = One or more errors occurred. (Cannot assign requested address var/run/iotedge/mgmt.sock)
```

## Steps to Reproduce
Run `iotedge check` on an Nvidia Jetson NX Devkit.

### Output of `iotedge check`

<details>
<summary>Click here</summary>

```
√ config.yaml is well-formed - OK
√ config.yaml has well-formed connection string - OK
√ container engine is installed and functional - OK
√ config.yaml has correct hostname - OK
× config.yaml has correct URIs for daemon mgmt endpoint - Error
    One or more errors occurred. (Cannot assign requested address var/run/iotedge/mgmt.sock)
        caused by: docker returned exit code: 1, stderr = One or more errors occurred. (Cannot assign requested address var/run/iotedge/mgmt.sock)
√ latest security daemon - OK
√ host time is close to real time - OK
√ container time is close to host time - OK
‼ DNS server - Warning
    Container engine is not configured with DNS server setting, which may impact connectivity to IoT Hub.
    Please see https://aka.ms/iotedge-prod-checklist-dns for best practices.
    You can ignore this warning if you are setting DNS server per module in the Edge deployment.
‼ production readiness: certificates - Warning
    The Edge device is using self-signed automatically-generated development certificates.
    They will expire in 89 days (at 2021-02-08 18:43:09 UTC) causing module-to-module and downstream device communication to fail on an active deployment.
    After the certs have expired, restarting the IoT Edge daemon will trigger it to generate new development certs.
    Please consider using production certificates instead. See https://aka.ms/iotedge-prod-checklist-certs for best practices.
‼ production readiness: container engine - Warning
    Device is not using a production-supported container engine (moby-engine).
    Please see https://aka.ms/iotedge-prod-checklist-moby for details.
‼ production readiness: logs policy - Warning
    Container engine is not configured to rotate module logs which may cause it run out of disk space.
    Please see https://aka.ms/iotedge-prod-checklist-logs for best practices.
    You can ignore this warning if you are setting log policy per module in the Edge deployment.
‼ production readiness: Edge Agent's storage directory is persisted on the host filesystem - Warning
    The edgeAgent module is not configured to persist its /tmp/edgeAgent directory on the host filesystem.
    Data might be lost if the module is deleted or updated.
    Please see https://aka.ms/iotedge-storage-host for best practices.
× production readiness: Edge Hub's storage directory is persisted on the host filesystem - Error
    Could not check current state of edgeHub container
        caused by: docker returned exit code: 1, stderr = Error: No such object: edgeHub

Connectivity checks
-------------------
√ host can connect to and perform TLS handshake with IoT Hub AMQP port - OK
√ host can connect to and perform TLS handshake with IoT Hub HTTPS / WebSockets port - OK
√ host can connect to and perform TLS handshake with IoT Hub MQTT port - OK
√ container on the default network can connect to IoT Hub AMQP port - OK
√ container on the default network can connect to IoT Hub HTTPS / WebSockets port - OK
√ container on the default network can connect to IoT Hub MQTT port - OK
√ container on the IoT Edge module network can connect to IoT Hub AMQP port - OK
√ container on the IoT Edge module network can connect to IoT Hub HTTPS / WebSockets port - OK
√ container on the IoT Edge module network can connect to IoT Hub MQTT port - OK

16 check(s) succeeded.
5 check(s) raised warnings.
2 check(s) raised errors.

```
</details>

### Device Information
* Host OS [e.g. Ubuntu 16.04, Ubuntu 18.04, Windows IoT Core]: Ubuntu 18.04
* Architecture [e.g. amd64, arm32, arm64]: arm64
* Container OS [e.g. Linux containers, Windows containers]: Linux containers

### Runtime Versions
* iotedged [run `iotedge version`]: 1.0.10.1
* Edge Agent [image tag (e.g. 1.0.0)]: 1.0
* Edge Hub [image tag (e.g. 1.0.0)]: 1.0
* Docker/Moby [run `docker version`]: 

```
Client:
 Version:           18.09.7
 API version:       1.39
 Go version:        go1.10.1
 Git commit:        2d0083d
 Built:             Fri Aug 16 14:20:24 2019
 OS/Arch:           linux/arm64
 Experimental:      false

Server:
 Engine:
  Version:          18.09.7
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.10.1
  Git commit:       2d0083d
  Built:            Wed Aug 14 19:41:23 2019
  OS/Arch:          linux/arm64
  Experimental:     false
```

## Logs
N/A

## Additional Information
N/A

**Comment by ancaantochi** [comment]
this bug will be fixed with the next 1.0.10 patch.

**Comment by veyalla** [comment]
1.0.10.2 has been released with the fix!
